README.md

```diff
@@ -45,12 +45,16 @@ configuration options:
 * Please configure the `rcrc` file if you'd like to make personal
   overrides in a different directory
 
-You can safely run `rcup` multiple times to update:
+
+Update
+------
+
+From time to time you should pull down any updates to these dotfiles, and run
 
     rcup
 
-You should run `rcup` after pulling a new version of the repository to symlink
-any new files in the repository.
+to link any new files and install new vim plugins. You can safely run `rcup`
+multiple times, so update early and update often!
```

Comment:
I think in the change here we are losing a little bit of the imperative. Can we add to this to state that pulling updates and _not_ running `rcup` may result in problems because there may be missing files or plugins?